docker-compose.yml

```diff
@@ -90,7 +80,7 @@ services:
       - 'lappy_network'
 
   mongodb:
-    image: mongodb/mongodb-community-server
+    image: mongodb/mongodb-community-server:4.4.18-ubuntu2004
```

Comment:
why we need to use old version? I m not sure that all me features will work on it

Reply:
ops, I forgot to remove it when I swapped from virtualbox. I will remove it